trino-s3-proxy/src/main/java/io/trino/s3/proxy/server/rest/RequestLoggingSession.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.s3.proxy.server.rest;
+
+import java.io.Closeable;
+
+public interface RequestLoggingSession
+        extends Closeable
+{
```

Comment:
Can we add a static noop implementation? When passing as an argument to a constructor `NoopRequestLoggingSession` is more descriptive than `() -> {}`.

```java
    public static final NoopRequestLoggingSession = () -> {};
```